pkgs/by-name/li/libglycin/package.nix

```diff
@@ -71,10 +73,33 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   passthru = {
-    updateScript = gnome.updateScript {
-      attrPath = "libglycin";
-      packageName = "glycin";
-    };
+    updateScript =
+      let
+        updateSource = gnome.updateScript {
+          attrPath = "libglycin";
+          packageName = "glycin";
+        };
+        updateLockfile = {
+          command = [
+            "sh"
+            "-c"
+            ''
+              PATH=${
+                lib.makeBinPath [
+                  common-updater-scripts
+                ]
+              }
+              update-source-version libglycin --ignore-same-version --source-key=cargoDeps.vendorStaging > /dev/null
+            ''
+          ];
+          # Experimental feature: do not copy!
```

Comment:
Commit message says

> Copied from gnome-user-share.

Source says

> # Experimental feature: do not copy!

Leaving comment just to confirm change

Reply:
Yes, I am aware of this line when copying, I assume @jtojnar's idea is to discourage using outside Nixpkgs.

https://github.com/NixOS/nixpkgs/blob/9137e671a2e2f5b46748ae21b21737b3f4194883/pkgs/common-updater/combinators.nix#L5-L7

https://github.com/NixOS/nixpkgs/blob/9137e671a2e2f5b46748ae21b21737b3f4194883/pkgs/common-updater/combinators.nix#L16-L22

The validation requires when exactly one script declaring `commit` feature (in this case `gnome.updateScript`), other scripts need to declare `silent`.

https://github.com/NixOS/nixpkgs/blob/49606d943aa634938c6b74f35125e6b7a2ba2bc7/pkgs/common-updater/combinators.nix#L127-L129

https://github.com/NixOS/nixpkgs/blob/49606d943aa634938c6b74f35125e6b7a2ba2bc7/pkgs/common-updater/combinators.nix#L138

Reply:
> Yes, I am aware of this line when copying, I assume @jtojnar's idea is to discourage using outside Nixpkgs.

Yeah, that was my intention.